examples/text_to_image/train_text_to_image_flax.py

```diff
@@ -272,10 +272,7 @@ def main():
     if args.dataset_name is not None:
         # Downloading and loading a dataset from the hub.
         dataset = load_dataset(
-            args.dataset_name,
```

Comment:
this is unrelated to this PR no?

Reply:
Otherwise, the styling fails. Can you make these changes in `main` directly?